app/styles/tools/_utilities.scss

```diff
@@ -355,6 +366,10 @@
 
     .pl-tiny#{_get-variant-name($breakpoint, '-')} { padding-left: _get-space('v', 'tiny'); }
     .pl-medium#{_get-variant-name($breakpoint, '-')} { padding-left: _get-space('v'); }
+
+    .position-relative#{_get-variant-name($breakpoint, '-')} { position: relative; }
+    .position-absolute#{_get-variant-name($breakpoint, '-')} { position: absolute; }
+    .position-fixed#{_get-variant-name($breakpoint, '-')} { position: fixed; }
   }
```

Comment:
@matusmarcin Raz si spomenul ze `!important` nechceme, nerozumiem preco. Niekedy to bez nich nejde. Ked potrebujem natvrdo overridnut nejaky element. Preto (zatial iba pre mt- classy) som musel pridat !important lebo mi to bez toho nebralo (long story, mozem ti ukazat ak chces). 

Tiez tu teda riesim zopar dalsich helperov aby sa dali pouzivat nezavisle pre jednotlive velkosti displejov.

Reply:
@mferak Nemam tu s tym velky problem, lebo sa mi zda, ze to tam neplaces (nepláceš) len tak nekoncepcne.

Vo vseobecnosti sa pouzivanie !important povazuje skor za anti-pattern, ajked samozrejme ludia sa na tom vedia pobit :) Tu napr. je aj nejaka argumentacia: https://uxengineer.com/css-specificity-avoid-important-css/

Kazdopadne, ja chcem byt trochy pragmaticky v tom a teda sa !important snazim nepouzivat pokial to je len mozne (niekedy aj za cenu dlhsieho sa hrabania), ale nie vzdy sa mi dari tomu vyhnut. Neplati zase ze za kazde pouzitie !important zomrie jedno maciatko 😄